Cake.DocumentDb/Cake.DocumentDb.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Cake.DocumentDb</id>
-    <version>4.0.7</version>
+    <version>4.0.8</version>
```

Comment:
Currently published package is 4.0.8 (https://www.nuget.org/packages/Cake.DocumentDb/) so this should be 4.0.9